index.html

```diff
@@ -391,14 +391,17 @@
 </div>
 <p>The <dfn>invoke idle callbacks algorithm</dfn>:</p>
 <ol>
+<li>Let <var>windows</var> be the <code>Window</code> object's list of <a href="https://www.w3.org/TR/html5/browsers.html#browsing-context">browsing context</a>'s <code>WindowProxy</code> objects that have a non-empty <a>list of runnable idle callbacks</a> (<var>runlist</var>).</li>
```

Comment:
WindowProxy objects don't have any sorts of lists of callbacks.

And a Window doesn't have any sort of list of WindowProxy objects associated with it, except its child windows, I suppose.